index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio Setup</title>
+</head>
+<body>
+     <main>
```

Comment:
- [ ] [OPTIONAL] To stick to the original design, Kindly fix your margins and background image 👍 

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![Screenshot 2023-06-01 145140](https://github.com/AbuTalha3/Portfolio-Setup/assets/57408419/8d700f40-1418-49de-b0b9-7188d8b9b89b)| |![Screenshot 2023-06-01 144940](https://github.com/AbuTalha3/Portfolio-Setup/assets/57408419/0de877b9-d8c7-4b88-9099-ba1641f0fba5)| |